chapters/annotations.tex

```diff
@@ -584,7 +611,7 @@ \subsection{Connection Restrictions}\label{connection-restrictions}
 
 A connector component declaration may have the following annotation:
 \begin{lstlisting}[language=modelica]
-annotation(mayOnlyConnectOnce = "message");
+/*literal*/ constant String mayOnlyConnectOnce;
```

Comment:
Default can be expressed with declaration equation:
```suggestion
/*literal*/ constant String mayOnlyConnectOnce = false;
```

Reply:
It's a string so false would be wrong. But could reformulate.

Reply:
I added a default and rephrased.